encrypt/encrypt.go

```diff
@@ -225,16 +227,34 @@ func Encrypt(args []string) error {
 		}
 	}()
 
-	// encrypt the input files
 	numFiles := len(files)
+
+	// create progress bar instance
+	p := mpb.New()
+
+	// encrypt the input files
 	for i, file := range files {
 		log.Infof("Encrypting file %v/%v: %s", i+1, numFiles, file.Unencrypted)
+		fileSize, _ := helpers.GetFileSize(file.Unencrypted)
+		bar := p.AddBar(fileSize,
+			mpb.PrependDecorators(
+				decor.Name(fmt.Sprintf("Encrypting %-12s", file.Unencrypted), decor.WCSyncWidth),
+			),
+			mpb.AppendDecorators(
+				decor.CountersKibiByte("% .1f / % .1f"),
+				decor.OnComplete(decor.Percentage(decor.WC{W: 5}), "done"),
+			),
+		)
```

Comment:
If you want to track the encryption progress this should be in the unexported function starting down on line 422.
You need to implement a progress tracker in combination with a teereader or multiwriter to achieve the goal.